### ISSUE TITLE
Empathy should take notes from player into account, when shadowing

### ISSUE THREAD

**Issue by Krycke** [Hanabi-Live/hanabi-live#2736]
When you are shadowing Alice, a card should change according to Alice' notes. If she is writing [f][b3] the card should morph into b3 with a light blue border around.

**Comment by Zamiell** [comment]
i dont think so, we want to keep notes separate

**Comment by Krycke** [comment]
Then maybe we should let notes from the shadowing player change the empathy, as it does a normal player. Now for example of you shadow a throw it in a hole game, and you write something on a stack. Nothing is happening, and you can't see the affects from the player you are showing either

**Comment by Zamiell** [comment]
every player who is shadowing is expected to keep their own copy of the notes, independent of what alice is doing.
thus i dont think it should affect empathy either

**Comment by Krycke** [comment]
But then if you are playing as a hydra head together with Alice. There is no way for you to make the blue border around a finessed card appear, which makes it hard for you to keep track of everything, as you need to scroll through all notes to see if a card is finessed or not. Why not let the cards be displayed to the shadowing player, as the notes describe them, the same way they do a normal player?

**Comment by Zamiell** [comment]
> There is no way for you to make the blue border around a finessed card appear

i don't follow, can't you just write [f]

> But then if you are playing as a hydra head together with Alice

many players use the shadow feature without playing hydra, to try and see if they can independently come to the same gameplay decision.
thus, adding in the functionality you describe will be destructive

**Comment by Zamiell** [comment]
> If you as a shadowing player types [f] on a note, the card don't change at
all. Not as a normal spectator either. That's the bug in pointing out.

i see

i would also like to see 1, and am currently surprised that it doesn't already work this way

**Comment by Krycke** [comment]
Yeah, I thought it was a feature, not to show any effect from the spectators notes. But then good, we agree. I will look into it.

**Comment by Zamiell** [comment]
maybe it is a feature, and i just dont remember the reasoning behind it
